embassy/Cargo.toml

```diff
@@ -14,6 +14,7 @@ time-tick-32768hz = ["time"]
 time-tick-1000hz = ["time"]
 time-tick-1mhz = ["time"]
 
+defmt = ["dep:defmt", "embassy-macros/defmt"]
```

Comment:
Note that this apparently requires `-Z namespaced-features`.

Reply:
In the past we've decided to not use `namespaced-features`. the problem is that it requires enabling it in end user's `.cargo/config.toml`. There's a similar situation already with `embassy-traits`.

Maybe we can add a `Spawner::must_spawn()` method so the panic is inside `embassy` so it can simply use the `defmt` featurE?

Reply:
This sounds plausible.